notebooks/CubevizExample.ipynb

```diff
@@ -204,7 +204,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "subset1_spec1d = spectra_dict['jw02732-o004_t004_miri_ch1-shortmediumlong_s3d.fits[SCI] (Subset 1)']\n",
+    "subset1_spec1d = spectra_dict['Spectrum (Subset 1, sum)']\n",
```

Comment:
This makes me wonder for when Cubeviz allows loading multiple FLUX cubes, won't "Spectrum" be too ambiguous?

Reply:
Yes, we would need to revisit the automatic labeling to include the label of the input cube (which could get long).  If we think that is on the horizon, I agree that it might be worth considering that now.

Reply:
Maybe can ask PO.